src/components/product-menu/__tests__/product-menu-test-cases.js

```diff
@@ -0,0 +1,15 @@
+import { ProductMenu } from '../product-menu';
+
+const testCases = {};
+const noRenderCases = {};
```

Comment:
We could probably trim out `noRenderCases` when there aren't any, so there's less boilerplate.

Reply:
👍 i don't think we have any of these set up now, so I added a reminder for myself to add something to the README - https://github.com/mapbox/frontend-documentation/issues/22#issuecomment-405339252.